functions/processQueueItem.js

```diff
@@ -30,7 +26,7 @@ module.exports.handler = async function({ ReceiptHandle, Body }) {
     });
 
     const upstreamServiceResponse = await request.post({
-      url: `${UPSTREAM_SERVICE_URL}?enhance`,
+      url: `${upstreamServiceUrl}?enhance`,
```

Comment:
Moved the upstream service URL into the accept endpoint, so it can construct the mock URL from request environment